src/meta/protos/proto/metadata.proto

```diff
@@ -18,6 +18,13 @@ package databend_proto;
 
 import "datatype.proto";
 
+message ColumnIdVector {
+  uint64 ver = 100;
+  uint64 min_reader_ver = 101;
+
+  repeated uint32 column_ids = 1;
+}
+
```

Comment:
If `ColumnIdVector` won't be decoded as a standalone message, `ver` and `min_reader_ver` can be removed.